ctapipe/tools/process.py

```diff
@@ -74,9 +74,7 @@ class ProcessorTool(Tool):
         ("o", "output"): "DataWriter.output_path",
         ("t", "allowed-tels"): "EventSource.allowed_tels",
         ("m", "max-events"): "EventSource.max_events",
-        "energy-regressor": "ShowerProcessor.EnergyRegressor.load_path",
-        "particle-classifier": "ShowerProcessor.ParticleClassifier.load_path",
-        "disp-reconstructor": "ShowerProcessor.DispReconstructor.load_path",
+        "reconstructor": "ShowerProcessor.reconstructor_paths",
```

Comment:
Have you considered how will this work with non-ML reconstructors or those that  don't have input files/serialized versions?
* ImPACT (not  a ML reco, but it does have a model as input, so we could just implement `read()` and it should work)
* Model3D (no input path needed - how would you specify it to be run?)

Reply:
Yes, those are still configuratble via the name list `reconstructor_types` as before. We just *additionally* have the list of paths.

Reply:
Thanks, I missed that..  One issue is that reconstructors listed in `reconstuctor_types` are always applied before those listed in `reconstructor_paths`, which then limits the user choice in order (i.e. Model3D, which needs a preliminary geometry as a seed can only get that seed from a reconstructor listed in the types list).   It's a little bit arbitrary.  Though I had trouble coming up with something better. Originall I thought to just have the  input file come in the config for the specific reconstructor itself:

```yaml
ShowerProcessor:
    reconstuctors: 
        - HillasReconstructor
        - EnergyRegressor
        - HillasIntersection
        - ImPACTReconstructor
        - Model3DReconstructor

ImPACTReconstructor:
    model_path: "impact-model.h5"
    geometry_seed: HillasReconstructor
    energy_seed: EnergyRegressor

EnergyRegressor:
    model_path: "energy-regressor-v1.h5"
...
```

Though in that case, you can't have multiply `EnergyRegressors` trained on different parameters, which could be a nice feature (e.g. an EnergyRegressor that uses the Hillas geometry, and another that uses the Model3D geometry. 

So I guess for now, having these two separate (but somewhat equal) lists are ok.

Reply:
Yes, that was the other option, however that required quite "hacky" or complex code in both the `__init__` methods of such reconstructors that support reading from a pickled (or another) path and to enable the aliases in the tools.

We could leave out  aliases, that would mean having something like `--reconstructor-types EnergyRegressor --EnergyRegressor.load_path=path/to/energy.pkl` or doing it via config files. 

I thought the two lists was the better solution for now. Not by much though, I could also re-add the complex load coad in the `__init__` method.

Reply:
> We could leave out aliases, that would mean having something like --reconstructor-types EnergyRegressor --EnergyRegressor.load_path=path/to/energy.pkl or doing it via config files.

That actually is cleaner I think.  I can't see that we will be using the command-line to change the reconstruct list very often so it doesn't have to be pretty — it would really be something you configure once via a config file.

Reply:
I tend to agree, so I will probably revert the adding of the path here and add back to `load_path` magic to the reconstructors that need it. Will convert this  back to draft state

Reply:
I changed that part back and adapted the CLI of `process` accordingly. It is more flexible but not as nice as before. But I guess we will mostly use "apply-models" or config files, so maybe consistency is more important than nice cli options.